docs/configuration.asciidoc

```diff
@@ -169,12 +169,12 @@ Use this setting in case the APM Server requires a token (e.g. APM Server in Ela
 [options="header"]
 |============
 | Default                 | Type
-| `http://localhost:8200` | List
+| `http://localhost:8200` | String
```

Comment:
Added a comment on this one: https://github.com/elastic/apm-agent-dotnet/pull/375/files#r304312327